pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -48,4 +49,8 @@ type DeschedulerConfiguration struct {
 
 	// EvictLocalStoragePods allows pods using local storage to be evicted.
 	EvictLocalStoragePods bool `json:"evictLocalStoragePods,omitempty"`
+
+	// Logging specifies the options of logging.
+	// Refer [Logs Options](https://github.com/kubernetes/component-base/blob/master/logs/options.go) for more information.
+	Logging componentbaseconfig.LoggingConfiguration `json:"logging,omitempty"`
```

Comment:
I was getting a complaint from `update-generated-conversions.sh` without that. Not sure if it's really needed though.